### PR TITLE
Ssh keys

### PR DESCRIPTION
**Pull request by obino**
Don't override the existing keys (and remove duplicates entries).

### CODE REVIEW

File: AppController/djinn.rb [hunk 1]
```diff
@@ -3590,23 +3590,28 @@ def copy_encryption_keys(dest_node)
     Djinn.log_info("Copying SSH keys to node at IP address #{ip}")
     ssh_key = dest_node.ssh_key
 
+    # Commands used in the cloud environment to set the root access.
+    options = "-o StrictHostkeyChecking=no -o NumberOfPasswordPrompts=0"
+    set_authorized_keys = "sudo sort -u ~#{user_name}/.ssh/authorized_keys /root/.ssh/authorized_keys -o #{CONFIG_FILE_LOCATION}/ssh_access"
+    enable_root_login = "sudo sed -n '/.*Please login/d; w/root/.ssh/authorized_keys' #{CONFIG_FILE_LOCATION}/ssh_acces"
```
Comment (cdonati) [inline AppController/djinn.rb]: Is `ssh_acces` a typo? Also, would it make sense to remove that file afterwards (like in tools)?

Comment (obino) [inline AppController/djinn.rb]: Nope, it was me not knowing how to get a tempfile name in ruby :). The new patch should address this.

File: AppController/djinn.rb [hunk 2]
```diff
@@ -3590,23 +3591,32 @@ def copy_encryption_keys(dest_node)
     Djinn.log_info("Copying SSH keys to node at IP address #{ip}")
     ssh_key = dest_node.ssh_key
 
+    # Commands used in the cloud environment to set the root access.
+    options = "-o StrictHostkeyChecking=no -o NumberOfPasswordPrompts=0"
+    temp_file = Dir::Tmpname.make_tmpname('/tmp/appscale-', nil)
+    set_authorized_keys = "sudo sort -u ~#{user_name}/.ssh/authorized_keys /root/.ssh/authorized_keys -o #{temp_file}"
```
Comment (cdonati) [inline AppController/djinn.rb]: Has `user_name` been defined yet?

Comment (cdonati) [inline AppController/djinn.rb]: Does `user_name` need to be defined before this line?

**Comment by obino** [comment]
OCD run is https://ocd.appscale.com:8080/job/Daily%20Build/95/

**Pull request merged**